### DISCUSSION TITLE
Buen dia! Este se puede utilizar en woocommerce?

### DISCUSSION THREAD

**Discussion by matijdk** [decidir/sdk-php-v2#30] [General]
Perdon la consulta, pero estoy buscando y no encuentro el plugin para woocomeerce

**Comment by wadeturrism** [comment]
hola buen dia, esta disponible el plugin para de woocomerce pero tendrias que realizar el alta en la plataforma y descargarlo.

**Comment by matijdk** [comment]
Muchas gracias! Viene aca de hecho porq cuando entro por la plataforma (ya estoy registrado y soy cliente) no puedo acceder a un link donde este para descargar el plugin que necesito. Si necesitas puedo pasarte mis dato de usuario de la plataforma asi podes corroborarlos
________________________________
De: Wilfredo De Turris ***@***.***>
Enviado: viernes, 10 de junio de 2022 12:31
Para: payway-ar/sdk-php-ventaonline ***@***.***>
Cc: matijdk ***@***.***>; Author ***@***.***>
Asunto: Re: [payway-ar/sdk-php-ventaonline] Buen dia! Este se puede utilizar en woocommerce? (Discussion #30)


hola buen dia, esta disponible el plugin para de woocomerce pero tendrias que realizar el alta en la plataforma y descargarlo.

**Comment by wadeturrism** [comment]
Pasame tu mail y lo seguimos por esa via

**Comment by matijdk** [comment]
dale, muchas gracias! el mail es ***@***.***
________________________________
De: Wilfredo De Turris ***@***.***>
Enviado: viernes, 10 de junio de 2022 13:34
Para: payway-ar/sdk-php-ventaonline ***@***.***>
Cc: matijdk ***@***.***>; Author ***@***.***>
Asunto: Re: [payway-ar/sdk-php-ventaonline] Buen dia! Este se puede utilizar en woocommerce? (Discussion #30)


Pasame tu mail y lo seguimos por esa via

**Comment by wadeturrism** [comment]
![Imagen1](https://user-images.githubusercontent.com/15057557/173088207-b642589c-a2e9-475a-8d0e-827695862712.png)

Deberias ver la opcion Appstore y de alli descargarlo, sino pasame el id de tu tienda o usario para validar.

**Reply by matijdk** [reply]
dale, es ambarvioletajoyeria@hotmail.com el usuario. no se si necesitas algun dato mas decime y te paso asi me envias ese link del plugin, gracias

**Reply by matijdk** [reply]
La verdad es que dentro de la web no encontre por ningun lado esa opcion de app!

**Comment by matijdk** [comment]
Hola otra vez! Te sirvieron los datos que te pase, el email? o necesitas algo mas?? Perdon pero tengo que poner la pasarela de pago en la web ya que todopago no va a funcionar mas! Pedime cualquier dato que necesites, gracias
________________________________
De: Wilfredo De Turris ***@***.***>
Enviado: viernes, 10 de junio de 2022 14:35
Para: payway-ar/sdk-php-ventaonline ***@***.***>
Cc: matijdk ***@***.***>; Author ***@***.***>
Asunto: Re: [payway-ar/sdk-php-ventaonline] Buen dia! Este se puede utilizar en woocommerce? (Discussion #30)


[Imagen1]<https://user-images.githubusercontent.com/15057557/173088207-b642589c-a2e9-475a-8d0e-827695862712.png>

Deberias ver la opcion Appstore y de alli descargarlo, sino pasame el id de tu tienda o usario para validar.

**Reply by wadeturrism** [reply]
Hola, a que email puedo contactarte? y pudieras pasarme la pagina a la que estas ingresando?

**Comment by matijdk** [comment]
https://www.ambarvioletajoyeria.com/
[https://www.ambarvioletajoyeria.com/wp-content/uploads/2022/05/20220530_120935_resized-290x290.jpg]<https://www.ambarvioletajoyeria.com/>
Ambar Violeta Joyeria<https://www.ambarvioletajoyeria.com/>
Venta de productos en plata, oro y acero de máxima calidad, siempre al mejor precio, ofreciendo descuentos en pagos en efectivo y promociones con todas las tarjetas. Nos caracterizamos por brindar una excelente atención a nuestros clientes. Somos una empresa familiar, especializada en el rubro joyería desde hace más de 30 años.
www.ambarvioletajoyeria.com

***@***.***
________________________________
De: Wilfredo De Turris ***@***.***>
Enviado: lunes, 13 de junio de 2022 19:34
Para: payway-ar/sdk-php-ventaonline ***@***.***>
Cc: matijdk ***@***.***>; Author ***@***.***>
Asunto: Re: [payway-ar/sdk-php-ventaonline] Buen dia! Este se puede utilizar en woocommerce? (Discussion #30)


Hola, a que email puedo contactarte? y pudieras pasarme la pagina a la que estas ingresando?
